pyo3-ffi/src/object.rs

```diff
@@ -147,13 +147,13 @@ pub struct PyVarObject {
 // skipped private _PyVarObject_CAST
 
 #[inline]
-#[cfg(not(all(PyPy, Py_3_10)))]
+#[cfg(not(all(GraalPy, PyPy, Py_3_10)))]
```

Comment:
This looks wrong, it's impossible to have graalpy && pypy.

Probably meant to be

```suggestion
#[cfg(not(any(GraalPy, all(PyPy, Py_3_10))))]
```